ci_framework/roles/edpm_deploy_baremetal/tasks/main.yml

```diff
@@ -196,6 +196,20 @@
             "path": "/spec/nodeTemplate/ansible/ansibleVars/edpm_container_registry_insecure_registries",
             "value": [{{ content_provider_registry_ip }}:5001]}]'
 
+- name: Patch OpenStackDataPlaneNodeSet resource to update bootstrap packages
+  when: not cifmw_edpm_deploy_baremetal_dry_run
+  environment:
+    KUBECONFIG: "{{ cifmw_openshift_kubeconfig }}"
+    PATH: "{{ cifmw_path }}"
+  ansible.builtin.command:
+    cmd: >-
+      oc patch openstackdataplanenodeset openstack-edpm-ipam
+      -n {{ cifmw_install_yamls_defaults['NAMESPACE'] }}
+      --type json
+      -p '[{"op": "add",
+            "path": "/spec/nodeTemplate/ansible/ansibleVars/edpm_bootstrap_command",
+            "value": "sudo dnf -y update"}]'
+
```

Comment:
this might mask actrul reall issue with edpm-ansible.
so I'm not sure its correct to do this in the ci-framework

i think adding a dnf update to the bootstrap role in edpm-ansibele is actually the correct fix.

Reply:
thank you @SeanMooney , Sounds good. I will update the same in edpm-ansible.

Reply:
We will merge this one to unblock integration line and then we will revert back once edpm-ansible pr is ready.

Reply:
@raukadah please revert this one after you get edpm-ansible pr merged. Thanks